File: ts/src/integrations/openai/sink.ts
/**
 * Re-export of the shared sink primitives.
 *
 * Kept for backward compatibility with existing internal imports within this
 * package. New integrations should import directly from
 * `autoctx/integrations/_shared`.
 */
export { FileSink } from "../_shared/sink.js";
export type { TraceSink, FileSinkOptions } from "../_shared/sink.js";

File: autocontext/src/autocontext/integrations/openai/_sink.py
"""Re-export of the shared sink primitives.

Kept for backward compatibility with existing internal imports within the
``autocontext.integrations.openai`` package. New integrations should import
directly from ``autocontext.integrations._shared``.
"""
from autocontext.integrations._shared.sink import FileSink, TraceSink

__all__ = ["FileSink", "TraceSink"]
